components/brave_shields/browser/ad_block_service_helper.h

```diff
@@ -14,6 +15,9 @@
 
 namespace brave_shields {
 
+void AdBlockServiceDomainResolver(const char* host, uint32_t* start,
```

Comment:
I don't see any reason to make this public, just create an anonymous namespace method in `AdBlockService`